### ISSUE TITLE
Scanner can't read wanderer's name in russian

### ISSUE THREAD

**Issue by Daniil-pizza** [Andrewthe13th/Inventory_Kamera#470]
### Current Behavior

![InventoryKamera_gUKgLKP0Ja](https://github.com/Andrewthe13th/Inventory_Kamera/assets/122693904/3d4c5f01-57b3-485e-bfff-0a64341428b5)
![GenshinImpact_So2tjj3gTq](https://github.com/Andrewthe13th/Inventory_Kamera/assets/122693904/29c2e709-9c58-4aff-88cb-5059d25ef290)

### Expected Behavior

The scanner continues to scan

### Reproducing a Behavior

Just start scanning characters 

### Device OS

Windows 10

### Genshin Impact Version

4.0

### Inventory Kamera Version

1.3.10

### Screen resolution, screen mode, and UI scale

full screen 1920x1080

### Additional notes and remarks

when scan comes to wanderer, it just stops on attributes
and suddenly solution suddenly is quite simple, i just cleared field with wanderer's name
attaching logs tho scanner haven't said it was an error

### Evidence

[InventoryKamera.debug.log](https://github.com/Andrewthe13th/Inventory_Kamera/files/12501530/InventoryKamera.debug.log)
[InventoryKamera.log](https://github.com/Andrewthe13th/Inventory_Kamera/files/12501531/InventoryKamera.log)

### Preflight Checklist

- [X] I have checked for similar issues in the issue page, and I have not found any similar to mine.
- [X] I am using the latest version of the scanner.
- [ ] I tried setting the scanning delay to maximum, and it still doesn't work.
- [ ] I have attached the video recording to the bug report.
- [ ] I have attached the error log to the bug report.
- [X] I am available to later explains the issue.
- [X] I have disabled all screen filtering software or plugins (i.e. HDR, Image Sharpening) while scanning.
- [X] I use default in-game keybindings or have configured keybindings for the scanner.

**Comment by man90es** [comment]
This isn't a bug. No languages except for English are currently supported, and adding support for them [is not planned](https://github.com/Andrewthe13th/Inventory_Kamera?tab=readme-ov-file#what-languages-does-inventory-kamera-support).

**Comment by Jeinara** [comment]
I don't know if this makes any sense or not, but it's still worth a try. The Tesseract recognizes all non-Latin characters... somehow. For example, my traveler's name Люми, parsed as jllumn. I saw it in the output window. And I guessed that Сенко could be Cenko, and I succeeded. The parser stopped thinking that Сенко is Cyno. I guess that even Chinese characters are parsed by the tesseract into some Latin letters. Perhaps if we had a screenshot insertion window for Wanderer's name in kamera interface somewere and parsed output of that, this feature would make life a little easier.
